source/SensorDetailsView.mc

```diff
@@ -74,37 +74,26 @@ class SensorDetailsDelegate extends WHIMBehaviorDelegate {
 
     function timerCallback() {
         mTimer.stop();
-        var menu = new WatchUi.Menu2( {:title => "ID"} );
-        menu.addItem(
-            new WatchUi.MenuItem(
-                Rez.Strings.text_command_rename,
-                "",
-                Rez.Strings.text_id_rename,
-                null
-            )
-        );
-        menu.addItem(
-            new WatchUi.MenuItem(
-                Rez.Strings.text_command_erase,
-                "",
-                Rez.Strings.text_id_erase,
-                null
-            )
-        );
-        WatchUi.switchToView(menu, new CommandMenuDelegate(), WatchUi.SLIDE_UP);
+        WatchUi.switchToView(initializeMenu(), new CommandMenuDelegate(), WatchUi.SLIDE_BLINK);
```

Comment:
Beautiful. I was going to leave a suggestion on the first commit to create a helper method here but I see you already have!